lib/fluent/output.rb

```diff
@@ -98,7 +98,7 @@ class OutputThread
     def initialize(output)
       @output = output
       @finish = false
-      @next_time = Engine.now + 1.0
+      @next_time = Time.now.to_f + 1.0
```

Comment:
float + float is faster than time + float, and time < float results in error

```
irb(main):001:0> Time.now < Time.now.to_f
ArgumentError: comparison of Time with 1427512006.7472174 failed
        from (irb):1:in `<'
```

but I think we can change it to time object here. let me try.